java.asciidoc

```diff
@@ -0,0 +1,298 @@
+= Pony for Java
```

Comment:
So which one is correct? I can try it out to be sure. I think the single `=` was accepted, but maybe not optimal.